src/utils/rfqm_health_check.ts

```diff
@@ -1,3 +1,10 @@
+import { Producer } from 'sqs-producer';
+
+const SQS_QUEUE_SIZE_DEGRADED_THRESHOLD = 5; // More messages sitting in queue than this will cause a DEGRADED issue
```

Comment:
I would make these numbers less aggressive to start:

e.g. 10 degraded, 20 failed

Reply:
fin